lib/migration_helper.rb

```diff
@@ -17,7 +17,7 @@ def rename old_name, new_name
       if name !~ /\+/   # no junctions
         new_name = name.gsub(cap_old, cap_new).gsub(down_old, down_new)
         count += 1
-        Card.find(id).update_attributes! :name=>new_name, :update_referencers=>true, :silent_change=>true
+        Card.find(id).update_attributes! :name=>new_name, :update_referers=>true, :silent_change=>true
```

Comment:
Line is too long. [102/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.